.gitignore

```diff
@@ -11,3 +11,7 @@ _projects/
 # Ignore build and release directories
 build/
 releases/
+# Generated file
+plugins/plugins.go
+*.etcd
```

Comment:
After make, I ran ./build/glusterd2 it generated above files in that directory itself. will remove these ignores